packages/client/src/app/pages/forms/edit/builder/tabs/layout-editor/property-editor/form-controls/options-editor/custom-options.styles.ts

```diff
@@ -0,0 +1,286 @@
+          text-align: center;
+          width: 20px !important;
+          background-color: #ffffff;
+        }
+
+        &:nth-last-child(3) {
+          width: 25px !important;
+        }
+
+        &:last-child {
+          border-right: 0;
+        }
+
+        input[type='checkbox'] {
+          padding: 0;
+          margin: -2px 0 0 0;
+          vertical-align: middle;
+        }
+      }
+
+      &:last-of-type {
+        td {
+          &:first-child {
+            border-bottom-left-radius: 3px;
+          }
+
+          &:last-child {
+            border-bottom-right-radius: 3px;
+          }
+        }
+      }
+    }
+  }
+`;
+
+export const TableAddOption = styled(Table)`
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  border-left: 1px dashed rgba(0, 0, 0, 0.1);
+  border-right: 1px dashed rgba(0, 0, 0, 0.1);
+  border-bottom: 1px dashed rgba(0, 0, 0, 0.1);
+
+  tbody {
+    tr {
+      td {
+      }
+    }
+  }
+`;
+
+export const CheckboxWrapper = styled.div`
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  margin: 0 0 10px 0;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: flex-start;
+`;
+
+export const Input = styled.input`
+  margin: 0;
+  border: 0;
+  width: 100%;
+  outline: none;
+  --focus-ring: 0;
+  border-radius: 0;
+  line-height: normal;
+  box-sizing: border-box;
+  background-color: #ffffff;
+`;
+
+export const AddOptionButton = styled.button`
+  margin: 0;
+  border: 0;
+  width: 100%;
+  display: flex;
+  outline: none;
+  padding: 5px 0;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  &:focus {
+    border: 0;
+    outline: none;
+  }
+`;
+
+export const DeleteButton = styled.button`
+  margin: 0;
+  border: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  outline: none;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  &:focus {
+    border: 0;
+    outline: none;
+  }
+`;
+
+export const DragButton = styled.button`
+  border: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  outline: none;
+  margin: -1px 0 0 0;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  &:focus {
+    border: 0;
+    outline: none;
+  }
+`;
+
+export const PlusIcon = styled.div`
+  padding: 0;
+  width: auto;
+  display: flex;
+  margin: 0 5px 0 0;
+  vertical-align: middle;
+
+  &::after {
+    margin: 0;
+    padding: 0;
+    content: '\\271B';
+    font-weight: 800;
+  }
+`;
+
+export const DragIcon = styled.div`
+  padding: 0;
+  width: auto;
+  display: flex;
+  margin: 1px 0 0 0;
+  vertical-align: middle;
+
+  &::after {
+    margin: 0;
+    padding: 0;
+    content: '\\2723';
+    font-weight: 800;
+  }
+`;
+
+export const DeleteIcon = styled.div`
+  margin: 0;
+  padding: 0;
+  width: auto;
+  display: flex;
+  vertical-align: middle;
+
+  &::after {
+    margin: 0;
+    padding: 0;
+    content: '\\2715';
+    font-weight: 800;
+  }
+`;
```

Comment:
Lots of identical code that has only a thing or two differing.
You could create a base component and then extend them instead.

```suggestion
const BaseIcon = styled.div`
  display: flex;
  vertical-align: middle;
  
  margin: 1px 0 0 0;

  &::after {
    font-weight: 800;
  }
`;

export const DragIcon = styled(BaseIcon)`
  &::after {
    content: '\\2723';
  }
`;

export const DeleteIcon = styled(BaseIcon)`
  &::after {
    content: '\\2715';
  }
`;
```